gems/pending/VMwareWebService/VimTypes.rb

```diff
@@ -95,6 +103,10 @@ def vimType=(val)
     @vimType = val.nil? ? nil : val.to_sym
   end
 
+  def vimBaseType
```

Comment:
Not directly related to the changes. But can all these methods be moved to a mixin?